MIGRATION.md

```diff
@@ -2,3 +2,15 @@
 - `identify` has been removed
 - `reset` has been removed
 - `createAlias` has been removed
+- `Package.product` has been changed from `SkuDetails` to `StoreProduct`. 
```

Comment:
might be more clear to name this file something like v4-migration? or to add a comment at the top specifying which version.

also, are we missing the PurchaserInfo rename in here?

Reply:
yes, will add that. good catch